server/src/main/java/org/apache/uniffle/server/buffer/ShuffleBufferManager.java

```diff
@@ -151,6 +152,19 @@ protected Entry<Range<Integer>, ShuffleBuffer> getShuffleBufferEntry(
   public ShuffleDataResult getShuffleData(
       String appId, int shuffleId, int partitionId, long blockId,
       int readBufferSize) {
+    return getShuffleData(
+        appId,
+        shuffleId,
+        partitionId,
+        blockId,
+        readBufferSize,
+        null
```

Comment:
Could we pass an empty collection?

Reply:
I think we'd better not. Because we rely on this to filter data, if it's empty, it means that all data should be filtered out.
Using the `null` to indicate this is invalid is reasonable.

Reply:
Ok, make sense.